spec/models/category_spec.rb

```diff
@@ -0,0 +1,3 @@
+RSpec.describe Category, type: :model do
+  it { is_expected.to validate_presence_of(:user_id) }
```

Comment:
this should also validate presence of name, as well as test the `self.link` method